### PR TITLE
added react.rb linkage

### PR DESCRIPTION
**Pull request by catmando**
Read.me update with suggested wording to for #433

### CODE REVIEW

**Comment by rmosolgo** [comment]
I don't really like this because it sends some false signals:

- it looks like the "reactjs" organization supports React.rb, but we don't have official support
- it looks like React.rb is bundled with react-rails, which it isn't. 

If another react-rails maintainer wants to support React.rb, feel free to say so. For my part, I'm busy enough with `react-rails` alone!

**Comment by catmando** [comment]
@rmosolgo Thanks for considering this... would it be possible to reword this to remove your concerns?  Possibly just stating clearly that it is a separate gem, with a separate OSS team.   I really believe this is a win-win.  There are a lot of ruby folks who might try react if they could stay in ruby, and the more people using react the better off we all are.

**Comment by rmosolgo** [comment]
I think it would be a good fit for a "Related Projects" list in the readme, but I don't think we should include the documentation here!

**Comment by rmosolgo** [comment]
We just got a "Related Projects" section here, feel free to update the React.rb section with another PR!
